docs/organizations/verified-domains.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: Verified Domains
+description: Learn about how to use verified domains for organizations
+---
+
+# Creating and Verifying a Domain
+
+Domains can be added and verified under an organization by administrators. In order to enable this feature, make sure to enable verified domains in the dashboard's _Organization Settings > Verified domains_. The only parameter needed to to create an organization domain is the `name`.
+
+Domains can be verified through an email verfication code sent to an email that matches the domain. If the administrator adding the domain already has a verified email using that domain in their account, the domain will be automatically verified.
+
+An instance may only have one verified domain of same name, and an organization may only have one domain of the same name (verified or unverified).
+
+# Enrollment Modes
+
+Once a domain is created and verified, an administrator can choose between two enrollment modes to enable new ways for users to join organizations.
+
+- `automatic invitations` - During sign up, a user will receive an invitation for the organization if their email's domain matches the verified domain. The user will join the organization if they accept the automatic invitation.
+- `automatic suggestions` - During sign up, a user will receive a suggestion for the organization if their email's domain matches the verified domain. The user can request to join, and an administrator must accept this request before the user joins the organization.
+
+# Usage
+
+```tsx copy
+const { organization, domains } = useOrganization();
```

Comment:
This `domains` attribute is not documented at all. And neither are its properties such as `prepareAfiilitationVerification`, `attemptAffiliationVerification`, etc.
https://clerk.com/docs/references/react/use-organization

```suggestion
const { organization, domains } = useOrganization();
```

Reply:
Thanks for the catch! Are you able to give me access to the Sanity docs so I can make those updates? If you prefer, I can also open a PR to migrate those docs to MDX, along with the updated `domains` attribute docs.